Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine
+
+RUN apk --no-cache add libreoffice ttf-dejavu
+
+WORKDIR /usr/app
+COPY ./ /usr/app
+COPY example.env /usr/app/.env
+RUN npm install
+
+EXPOSE 5227
```

Comment:
This is better left out. It doesn't mean anything (it's a comment, not a directive) and is likely to be left outdated if the default port is ever updated elsewhere.

Reply:
sure, I find it useful to see where the port is likely to be bound